packages/grafana-data/package.json

```diff
@@ -9,17 +9,18 @@
   ],
   "repository": {
     "type": "git",
-    "url": "http://github.com/grafana/grafana.git"
+    "url": "http://github.com/grafana/grafana.git",
+    "directory": "packages/grafana-data"
```

Comment:
what does this do?  what changes when you add directory?

Reply:
The link on npmjs.com will go to the package directory instead of the repo root